src/lib/sql/sql_pipeline.cpp

```diff
@@ -242,10 +242,13 @@ std::string SQLPipelineMetrics::to_string() const {
   std::vector<bool> query_plan_cache_hits;
 
   for (const auto& statement_metric : statement_metrics) {
-    total_translate_micros += statement_metric->translate_time_micros;
-    total_optimize_micros += statement_metric->optimize_time_micros;
-    total_compile_micros += statement_metric->compile_time_micros;
-    total_execute_micros += statement_metric->execution_time_micros;
+    total_translate_micros +=
```

Comment:
can these be nanos, too? May PC is too fast to reliably run the tests in release mode, because occasionally compilation or translation is faster than one us, which some tests rely on smh

Reply:
This is just when printing the times for the console. Didn't want to print ns there, because the numbers would be too big, but I'll change it to format_duration.